style.css

```diff
@@ -1854,3 +1845,46 @@ form button:focus {
 
   /* contact me section end */
 }
+
+/* mobile toggle section start */
+.mob-icon {
+  display: none !important;
+}
+
+.nav-ul .close-btn {
+  display: none;
+}
+
+.mobile-body {
+  position: fixed;
+  top: 72px;
+  background-color: #3e3c3c;
+  width: 100%;
+  height: 100%;
+  z-index: 99;
+  right: 100%;
+  display: none;
+  transition: all 3s;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+.mobile-body .nav-ul {
+  height: 100vh;
+  display: flex;
+  flex-direction: column;
+  padding: 50px;
+  gap: 20px;
+}
```

Comment:
- [ ] You did a good job 👏. However, there are some issues when there fixed your project would look as expected.

1.When we click on the hamburger menu, there is no close icon to close the menu. We have to scroll down to see the close icon, and there is a gap at the top. This, in my opinion, does not provide a good user experience for your site, and because a good user experience is critical for any piece of software, I would suggest you correct it so that visitors can navigate through your site more easily. For more information, please see 👇️

[Screencast from 06.02.2023 22:51:03.webm](https://user-images.githubusercontent.com/89228743/217109041-82fea87d-c761-400f-af2c-32feabf49b44.webm)

### To fix the issue please remove `position: relative;` you have at line `1573` in your `style.css` file.